CenterForOpenScience/ember-osf-web
# [EMB-439] Add head meta tags

## Purpose

Add head meta tags for social media sharing and facilitate discovery in Google Scholar and Zotero.

## Summary of Changes

- Add head tags to registries overview
- Add `keywords`, `author` to all collected head tags
- Add Zotero call to allow Zotero extension to pick up the meta tags.

## Side Effects

N/A

## Feature Flags

N/A

## QA Notes
-  Inspect head tags in the console elements tag and make sure they are correct compared to all legacy overview page.
- To inspect a particular tag: Open Graph (og) title, use:

```javascript
$('meta[property="og:title"]').attr('content')
```
## Ticket

[EMB-439](https://openscience.atlassian.net/browse/EMB-439)

# Reviewer Checklist
